README.md

```diff
@@ -32,19 +32,31 @@ This guide executes a Sonobuoy run on your cluster, and records the following re
 
 * *You should have `kubectl` installed.* If not, follow the instructions for [installing via Homebrew (MacOS)][6] or [building the binary (Linux)][7].
 
+* *You should have `docker` installed (minimum 17.05).* Docker images are used to generate the YAML examples, based on your cluster's RBAC settings.
```

Comment:
@ncdc I saw that on Docker multi-stage build page (https://docs.docker.com/engine/userguide/eng-image/multistage-build/), but if that version is a potential blocker for folks, I can repurpose the Dockerfile?